_posts/2018-02-23-this-week-in-rustfest-0-cfp-and-crabs.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "This week in RustFest 0: CfP, impl days, and little crabs"
+authors:
+  - dns2utf8
+  - hoverbear
+---
+
+With the next iteration of RustFest coming it means it's time for new `TwirfIterator`! Welcome to our first `Some(Post)`.
+
+# CFP
+
+[Our CFP](http://cfp.rustfest.eu/) is open again! Please consider submitting! Not everyone will be selected as a speaker, but taking time to think of a topic and put together a proposal can cause other, even unrelated, epiphanies!
```

Comment:
shall we use `https://cfp.rustfest.eu/` here it does exist and seeps to work

Reply:
fixed, thank you